src/main/java/rx/exceptions/OnErrorThrowable.java

```diff
@@ -150,6 +160,12 @@ private static String renderValue(Object value){
             if (value instanceof Enum) {
                 return ((Enum<?>) value).name();
             }
+
+            String pluggedRendering = RxJavaPlugins.getInstance().getErrorHandler().handleOnNextValueRendering(value);
```

Comment:
Perhaps the `ErrorHandler` should be statically referenced once at the top of this class?